### PR TITLE
Return of the PHPStan

### PR DESCRIPTION
**Pull request by kukulich**

### CODE REVIEW

File: src/NodeCompiler/CompileNodeToValue.php [hunk 1]
```diff
@@ -33,6 +33,9 @@ public function __invoke(Node $node, CompilerContext $context)
             return $this($node->expr, $context);
         }
 
+        /** @var Node\Expr $node */
+        $node = $node;
```
Comment (Ocramius) [inline src/NodeCompiler/CompileNodeToValue.php]: Isn't a declaration enough?

File: src/SourceLocator/Ast/Strategy/NodeToReflection.php [hunk 2]
```diff
@@ -36,6 +36,8 @@ public function __invoke(
         }
 
         if ($node instanceof Node\FunctionLike) {
+            /** @var Node\Stmt\ClassMethod|Node\Stmt\Function_|Node\Expr\Closure $node */
+            $node = $node;
```
Comment (Ocramius) [inline src/SourceLocator/Ast/Strategy/NodeToReflection.php]: Same: why is this being fone with an assignment? Also, should the instanceof check verify these 3 types?

File: src/NodeCompiler/CompileNodeToValue.php (hunk 1 above)
Comment (kukulich) [reply src/NodeCompiler/CompileNodeToValue.php]: Fixed with `@param Node\Stmt\Expression|Node\Expr $node` now.

File: src/SourceLocator/Ast/Strategy/NodeToReflection.php (hunk 2 above)
Comment (kukulich) [reply src/SourceLocator/Ast/Strategy/NodeToReflection.php]:
```
        if ($node instanceof Node\Stmt\ClassMethod
            || $node instanceof Node\Stmt\Function_
            || $node instanceof Node\Expr\Closure
        ) {
```

You're right, this is definitely better.

**Pull request merged**